lib/backburner/queue.ts

```diff
@@ -139,6 +129,16 @@ export default class Queue {
     return false;
   }
 
+  public push(target, method, args, stack) {
```

Comment:
Why is this moved around? Seems unrelated to the stated changes...

Reply:
just moved it `push`  and `pushUnique` next to each other :P, does it worth another PR

Reply:
ya, at least a separate commit so its easier to track why things were juggled around

Reply:
done